scripts/sanitize_metadata.py

```diff
@@ -157,6 +148,8 @@ def resolve_duplicates(metadata, strain_field, error_on_duplicates=False):
         formatter_class=argparse.ArgumentDefaultsHelpFormatter,
     )
     parser.add_argument("--metadata", required=True, help="metadata to be sanitized")
+    parser.add_argument("--metadata-id-columns", default=["Virus name", "strain", "name"], nargs="+", help="names of valid metadata columns containing identifier information like 'strain' or 'name'")
+    parser.add_argument("--database-id-columns", default=[], nargs="+", help="names of metadata columns that store external database ids for each record (e.g., GISAID, GenBank, etc.) that can be used to deduplicate metadata records with the same strain names.")
```

Comment:
If `--metadata-id-columns` defaults to the previous hardcoded value (modulo an ordering change¹), then is seems that `--database-id-columns` should too?

¹ Order does matter in the code, so probably should keep it the same as the previous hardcoded value if we're going to provide it as default?

Reply:
Ah! I see that it gains a default in a subsequent commit. Seems like the change got misplaced in the wrong commit.